meilisearch-types/src/error.rs

```diff
@@ -323,6 +324,10 @@ impl Code {
             DuplicateIndexFound => {
                 ErrCode::invalid("duplicate_index_found", StatusCode::BAD_REQUEST)
             }
+            ReceivePayloadErr => ErrCode::internal(
+                "receive_payload_internal_exceptions",
```

Comment:
This is a new error code that we never defined in the doc. I would say that we should use an already known one [like the _internal_ one](https://docs.meilisearch.com/reference/errors/error_codes.html#internal).

Reply:
Adjusting it to an internal code is no problem. I originally meant to show this better

Reply:
Yep sorry, currently, the way we see the error code is something along the lines of:
If there is an error code, there is a web page associated with it that explains why it happened and how you can solve your issue: https://docs.meilisearch.com/reference/errors/error_codes.html#error-codes

And since you can't do anything about an internal error like yours, we just output the internal error code and then tries to be explicit with the error message so when users encounter it, they can help us to understand where it happened.